R/exportToJson.R

```diff
@@ -20,22 +20,24 @@
 # @author Chris Knoll
 # @author Frank DeFalco
 
+
+# Run this definition of allReports when adding a new report
 # allReports <- c("CONDITION",
-#                "CONDITION_ERA",  
-#                "DASHBOARD", 
+#                "CONDITION_ERA",
+#                "DASHBOARD",
 #                "DATA_DENSITY",
 #                "DEATH",
 #                "DRUG",
 #                "DRUG_ERA",
 #                "HEEL",
 #                "OBSERVATION",
 #                "OBSERVATION_PERIOD",
-#                "PERSON", 
+#                "PERSON",
 #                "PROCEDURE",
 #                "VISIT",
 #                "MEASUREMENT",
 #                "META")
-# save(allReports,file="allReports.rda")
+# save(allReports,file="data/allReports.rda")

```

Comment:
@alondhe : was there a reason for placing this .rda file in a folder 'data'?  This RDA file is not associated to the export to JSON work (where we put the exported json files under a 'data' folder).  Was there some other reason for this change?  I think this caused the creation of another file (see other files added to the commit) in data/allReports.rda, while I think there's already a file calld allReports.rda elsewhere in the repository....